test/index.test.js

```diff
@@ -1,23 +1,33 @@
 const assume = require('assume');
-const Datastar = require('datastar');
+var { DynamoDB } = require('aws-sdk');
+var dynamo = require('dynamodb-x');
+var AwsLiveness = require('aws-liveness');
 const wrhsModels = require('warehouse-models');
-const config = require('./config');
 const sinon = require('sinon');
 
+process.env.AWS_ACCESS_KEY_ID = 'foobar';
+process.env.AWS_SECRET_ACCESS_KEY = 'foobar';
+const region = 'us-east-1';
+const endpoint = 'http://localhost:4569';
```

Comment:
why not repurpose `test/config.json` for these values, rather than having all config inline. Also I don't think these need to be forcefully set on `process.env`. So far I've been able to use `localstack` locally, just fine without these `env` values.

Reply:
Nvm travis is garbage and needs to env variables appearantly.

Reply:
@Swaagie yea localstack needs some kind of AWS_ACCESS_KEY_ID etc as env vars to fake the sdk to working properly